sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,103 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient. They can be expressed as
```

Comment:
please see the `binomial` docstring; I think a clearer statement can be made that indicates what this function calculates.

Reply:
I'll rewrite that.